app/routes/community/meetups/assets.js

```diff
@@ -0,0 +1,4 @@
+import Route from '@ember/routing/route';
```

Comment:
it's ok to delete this file 👍

Reply:
If i delete this then the assets routes has to be extended to under the `community/meetups/assets/index`.  That route is where the new assets page resides, the index page is for the main meetups route

Reply:
so you don't need to do anything if you delete this file. Deleting an **empty** route file will have no effect on the application because Ember will effectively create one for you in the background.

You only need to create a route file if you **do** something in that route 😄

Reply:
Oh shoot. that's true.. lemme fix and send a PR right away